internal/controller/cluster/access_control.go

```diff
@@ -164,7 +174,7 @@ func (r *SingleClusterReconciler) reconcileUsers(
 	desired map[string]asdbv1.AerospikeUserSpec,
 	current map[string]asdbv1.AerospikeUserSpec,
 	passwordProvider AerospikeUserPasswordProvider, client *as.Client,
-	adminPolicy as.AdminPolicy,
+	adminPolicy as.AdminPolicy, image string,
```

Comment:
Why do we need to pass the image as a func param?
All the spec info is already present in `r` receiver